src/pretix/plugins/sendmail/signals.py

```diff
@@ -116,21 +119,28 @@ def control_nav_import(sender, request=None, **kwargs):
     ]
 
 
-@receiver(signal=logentry_display)
-def pretixcontrol_logentry_display(sender, logentry, **kwargs):
-    plains = {
-        'pretix.plugins.sendmail.sent': _('Mass email was sent to customers or attendees.'),
-        'pretix.plugins.sendmail.sent.waitinglist': _('Mass email was sent to waiting list entries.'),
-        'pretix.plugins.sendmail.order.email.sent': _('The order received a mass email.'),
-        'pretix.plugins.sendmail.order.email.sent.attendee': _('A ticket holder of this order received a mass email.'),
-        'pretix.plugins.sendmail.rule.added': _('An email rule was created'),
-        'pretix.plugins.sendmail.rule.changed': _('An email rule was updated'),
-        'pretix.plugins.sendmail.rule.order.email.sent': _('A scheduled email was sent to the order'),
-        'pretix.plugins.sendmail.rule.order.position.email.sent': _('A scheduled email was sent to a ticket holder'),
-        'pretix.plugins.sendmail.rule.deleted': _('An email rule was deleted'),
-    }
-    if logentry.action_type in plains:
-        return plains[logentry.action_type]
+@log_entry_types.new('pretix.plugins.sendmail.sent', _('Mass email was sent to customers or attendees.'))
+@log_entry_types.new('pretix.plugins.sendmail.sent.waitinglist', _('Mass email was sent to waiting list entries.'))
+class SendmailPluginLogEntryType(EventLogEntryType):
+    pass
```

Comment:
I wonder if we even should use something like

```suggestion
log_entry_types.new('pretix.plugins.sendmail.sent', _('Mass email was sent to customers or attendees.'))(EventLogEntryType)
log_entry_types.new('pretix.plugins.sendmail.sent.waitinglist', _('Mass email was sent to waiting list entries.'))(EventLogEntryType)
```

explicitly here instead of creating many empty classes, what do you think?

Reply:
No, we need the empty classes here because this way they are declared in the correct package, so the registry can detect which plugin the LogEntryTypes come from. If you have a better solution for that though, I'd be very open, because I'm not a huge fan of the empty classes myself, but I couldn't think of a better way so far